packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -1579,3 +1581,27 @@ def get_current_work_item(self) -> WorkItem:
             Set Current Work Item    ${input}
         """
         return self.current
+
+    @keyword
+    def parse_work_item_from_email(self) -> dict:
+        """Parse and return a dictionary from the sent mail trigger.
+
+        Since a process can be started in Control Room by sending an e-mail, a body
+        in JSON format can be sent as well and this gets attached to the input work
+        item with the `rawEmail` payload variable. This keyword parses the content of
+        it and returns the original dictionary that was sent through the mail.
+
+        Example:
```

Comment:
this example is a bit odd to me

- why there is `Save Work Item`  ? 
- it is a bit unclear where `action` comes from, I suppose from email body. Maybe add multiline comment to show what email body actually looks like in this case ?

Reply:
or just 
# email body: {"action": "update"}
maybe ?

Reply:
Simplified it now:
- changed `action` -> `message`
- removed the save since it wasn't necessary
- added e-mail body example